src/build.py

```diff
@@ -1272,18 +1279,41 @@ def Test(self):
 
 
 def TestBare():
+  # Compile
   CompileLLVMTorture('s', TORTURE_S_OUT_DIR)
   CompileLLVMTorture('o', TORTURE_O_OUT_DIR)
+
+  # Link/Assemble
+  lld_out = LinkLLVMTorture(
+      name='lld',
+      linker=Executable(os.path.join(INSTALL_BIN, 'lld')),
+      fails=LLD_KNOWN_TORTURE_FAILURES,
+      input_pattern=os.path.join(TORTURE_O_OUT_DIR, '*.o'))
+
```

Comment:
Nit: Inconsistent spacing for lld, s2wasm, and wasm2wast. I have no strong opinion whether there should be spaces between or not, but they should match each other within a section.